src/devices/video/mc6845.cpp

```diff
@@ -1065,7 +1065,7 @@ void mc6845_device::device_start()
 	m_vsync = m_hsync = 0;
 	m_cur = 0;
 	m_line_counter = 0;
-	m_horiz_disp = m_vert_disp = 0;
+	m_horiz_disp = m_vert_disp = 0xff;
```

Comment:
This was needed, because with the new calculation at start up, the systems using the 6845 reports a bad screen resolution.

Reply:
This change was actually needed, so that the check on line 453 would consider the params invalid and not try to update screen, or calculate the resolution.